hookplugins/image_plugin.go

```diff
@@ -0,0 +1,20 @@
+package hookplugins
```

Comment:
you should update the plugin's document. 
https://github.com/alibaba/pouch/blob/master/docs/features/pouch_with_plugin.md